_posts/2019-11-21-vscode-quarkus-1.2.0.adoc

```diff
@@ -2,6 +2,7 @@
 layout: post
 title: 'Quarkus Tools for Visual Studio Code - 1.2.0 release'
 date: 2019-11-21
+tags: ide vscode
```

Comment:
just ide no? vscode is like kafka-streams, details?

Reply:
we can, not married to it.